backend/crypt/cipher.go

```diff
@@ -114,22 +117,65 @@ func (mode NameEncryptionMode) String() (out string) {
 	return out
 }
 
-// Cipher defines an encoding and decoding cipher for the crypt backend
```

Comment:
This comment didn't want to get deleted!

Reply:
Sorry about this. Fixed in 79817be